front/package-lock.json

```diff
@@ -11484,9 +11484,9 @@
       "link": true
     },
     "node_modules/@dust-tt/sparkle": {
-      "version": "0.2.298",
-      "resolved": "https://registry.npmjs.org/@dust-tt/sparkle/-/sparkle-0.2.298.tgz",
-      "integrity": "sha512-USEiF8fa6xZQFY+APFS2U2AaGMH1C0SHOROTuwtFPRzDL5M31/9RXHAgFNPCK9vGwVX3znpULdTjxzuUawbIoA==",
+      "version": "0.2.299-rc-1",
```

Comment:
LGTM, can you publish a new sparkle release ?
Also if you can do the search/replace on extension too 😇

Reply:
> LGTM, can you publish a new sparkle release ?

Yes def, I usually release a RC first to check if linter passes and once it does I publish sparkle. It avoids blocking sparkle for others 😄 

> Also if you can do the search/replace on extension too 

✅